ModuleNotFoundError: No module named 'social_django'

Was receiving the following error:

```
ModuleNotFoundError: No module named 'social_django'
Makefile:32: recipe for target 'migrate' failed
```

Perhaps it is the way that my computer was setup before, but upon searching I found that `pip install social-auth-app-django` should be run. I ran `pipenv run pip install social-auth-app-django` and this fixed the problem, but not sure if this is the correct way to go about things or not (maybe add to Pipfile).

#37 references getting rid of pipenv so this solution may not be relevant with changes to come, but wanted to mention it here so either setup can be fixed or setup documentation can mention the extra command.

--- a/Pipfile
+++ b/Pipfile
@@ -4,13 +4,13 @@ verify_ssl = true
 name = 'pypi'
 
 [packages]
-django = "~=2.1.5"
-mysqlclient = "~=1.3.13"
+django = "==2.1.9"
+mysqlclient = "==1.3.14"
 whitenoise = "==4.1.1"
 pillow = "==5.3.0"
 django-markdownx = "==2.0.27"
 gunicorn = "==19.9.0"
-fabric = "*"
+fabric = "==2.4.0"
 social-auth-app-django = "==3.1.0"
 
 [dev-packages]

--- a/Pipfile
+++ b/Pipfile
@@ -11,6 +11,7 @@ pillow = "==5.3.0"
 django-markdownx = "==2.0.27"
 gunicorn = "==19.9.0"
 fabric = "*"
+social-auth-app-django = "==3.1.0"
 
 [dev-packages]
 pre-commit = "==1.14.4"

--- a/requirements-dev.txt
+++ b/requirements-dev.txt
@@ -1,23 +1,24 @@
 -i https://pypi.python.org/simple
-aspy.yaml==1.1.2
+aspy.yaml==1.2.0
 atomicwrites==1.3.0
-attrs==18.2.0
-cfgv==1.4.0
+attrs==19.1.0
+cfgv==1.6.0
+colorama==0.4.1 ; sys_platform == 'win32'
 filelock==3.0.10
-identify==1.2.2
-importlib-metadata==0.8
+identify==1.4.1
+importlib-metadata==0.9
 livereload==2.6.0
-more-itertools==6.0.0 ; python_version > '2.7'
+more-itertools==7.0.0 ; python_version > '2.7'
 nodeenv==1.3.3
-pluggy==0.8.1
+pluggy==0.9.0
 pre-commit==1.14.4
-py==1.7.0
+py==1.8.0
 pytest-django==3.4.7
 pytest==4.3.0
-pyyaml==3.13
+pyyaml==5.1
 six==1.12.0
 toml==0.10.0
-tornado==5.1.1
+tornado==6.0.2
 tox==3.7.0
-virtualenv==16.4.0
+virtualenv==16.5.0
 zipp==0.3.3

--- a/requirements.txt
+++ b/requirements.txt
@@ -1,9 +1,32 @@
 -i https://pypi.python.org/simple
+asn1crypto==0.24.0
+bcrypt==3.1.6
+certifi==2019.3.9
+cffi==1.12.3
+chardet==3.0.4
+cryptography==2.6.1
+defusedxml==0.6.0 ; python_version >= '3.0'
 django-markdownx==2.0.27
-django==2.1.7
+django==2.1.8
+fabric==2.4.0
 gunicorn==19.9.0
-markdown==3.0.1
+idna==2.8
+invoke==1.2.0
+markdown==3.1
 mysqlclient==1.3.14
+oauthlib==3.0.1
+paramiko==2.4.2
 pillow==5.3.0
-pytz==2018.9
+pyasn1==0.4.5
+pycparser==2.19
+pyjwt==1.7.1
+pynacl==1.3.0
+python3-openid==3.1.0 ; python_version >= '3.0'
+pytz==2019.1
+requests-oauthlib==1.2.0
+requests==2.21.0
+six==1.12.0
+social-auth-app-django==3.1.0
+social-auth-core==3.1.0
+urllib3==1.24.2
 whitenoise==4.1.1

